core/src/cubos/core/data/file.cpp

```diff
@@ -444,8 +452,5 @@ File::~File()
 {
     if (this->destroyed && this->archive)
         if (!this->archive->destroy(this->id))
-        {
-            logError("Could not destroy file '{}', internal archive error", this->path);
-            abort();
```

Comment:
This `abort()` was removed, intended?

Reply:
Yeah, no need to abort() here, we can just log the error.